$NetBSD: patch-Makefile,v 1.3 2018/10/03 22:27:05 nia Exp $

Do not pick pkgsrc libtool on Darwin.
Use correct installation dirs.

--- Makefile.orig	2018-04-06 10:53:30.000000000 +0000
+++ Makefile
@@ -15,7 +15,7 @@ VERSION = $(shell cat VERSION)
 
 ifneq (, $(findstring darwin, $(OS)))
 JARSUFFIX=mac
-LIBTOOL ?= $(shell which libtool)
+LIBTOOL ?= /usr/bin/libtool
 endif
 ifneq (, $(findstring linux, $(OS)))
 JARSUFFIX=linux
@@ -149,7 +149,7 @@ endif
 INSTALLDIRINC = $(INSTALLPREFIX)/include/miniupnpc
 INSTALLDIRLIB = $(INSTALLPREFIX)/$(LIBDIR)
 INSTALLDIRBIN = $(INSTALLPREFIX)/bin
-INSTALLDIRMAN = $(INSTALLPREFIX)/share/man
+INSTALLDIRMAN = $(INSTALLPREFIX)/$(PKGMANDIR)
 PKGCONFIGDIR = $(INSTALLDIRLIB)/pkgconfig
 
 FILESTOINSTALL = $(LIBRARY) $(EXECUTABLES)
